ui-v2/app/templates/dc/services/show/services.hbs

```diff
@@ -1,12 +1,12 @@
 <div id="services" class="tab-section">
   <div role="tabpanel">
-  {{#if (gt gateway.Services.length 0)}}
+  {{#if (gt gateway.length 0)}}
```

Comment:
I mentioned this in my drive by review yesterday before I finished up, but just to mention that this is one of the things that looks strange to me, a singular named variable with a length. (btw I know this is kind of left over from where I'd recommended moving this to the new approach!!)

Reply:
Updated to be `gatewayServices`.